aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,47 +1,26 @@
 ---
 remote_endpoint: ~
-name: "v1.11"
+name: "v1.12"
 proposals:
   - name: step_1_increase_max_txn_gas
     metadata:
       title: "Increase max txn gas temporarily for framework upgrade"
       description: "Increase max txn gas temporarily for framework upgrade"
     execution_mode: MultiStep
     update_sequence:
-      - DefaultGasWithOverrideOld:
+      - DefaultGasWithOverride:
+          feature_version: 16
```

Comment:
This one is the correct. 16 is the gas version prior to 1.12. When we relax the execution limit we don't want to bump it accidentally.

Reply:
Just to clarify, by relaxing the execution limit, do you mean this override?
```
   overrides:
            - name: "txn.max_execution_gas"
              value: 3676000000
```

Reply:
Yes